Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -329,18 +299,30 @@ private void UpdateViewState()
             if (before != after)
             {
                 ViewStateChanged?.Invoke(this, after);
+                SetBackButtonVisibility(before);
```

Comment:
Shouldn't this just be `SetBackButtonVisibility(ViewState);`

Reply:
No, the SetBackButtonVisibility expects the previosState. Just cleaned up the method to make it more clear

Reply:
That doesn't make sense